test/unit/specs/components/common/PagePreferences.spec.js

```diff
@@ -17,7 +17,9 @@ describe("PagePreferences", () => {
     })
   })
 
-  it("has the expected html structure if connected", () => {
+  it("has the expected html structure if connected", async () => {
+    await wrapper.vm.$nextTick()
```

Comment:
Why do we need this here?

Reply:
$nextTick is added for the scrollbar plugin